website/content/docs/v0.7/Reference/configuration.md

```diff
@@ -1499,10 +1499,11 @@ Defaults to `false`.
 </div>
 <div class="dt">
 
-Specifies time (ntp) servers to use for setting system time.
+Specifies time (NTP) servers to use for setting the system time.
 Defaults to `pool.ntp.org`
 
-> Note: This parameter only supports a single time server
+
```

Comment:
This extra whitespace probably will be caught by markdown lint.

Reply:
It would, but I think the generated markdown files have an ignore clause.